tests/Application/config/packages/bitbag_sylius_shipping_export_plugin.yaml

```diff
@@ -1,2 +1,9 @@
 imports:
-    - { resource: "@BitBagSyliusShippingExportPlugin/Resources/config/config.yml" }
\ No newline at end of file
+    - { resource: "@BitBagSyliusShippingExportPlugin/Resources/config/config.yml" }
+
+sylius_resource:
+    resources:
+        bitbag.shipping_export:
+            classes:
+                model: Tests\BitBag\SyliusInPostPlugin\Application\src\Entity\ShippingExport
+                controller: BitBag\SyliusInPostPlugin\Controller\ShippingExportController
```

Comment:
Is this really needed? Do you override anything in ShippingExportController::exportSingleShipmentAction() or ShippingExportController::exportAllNewShipmentsAction() ? If yes, couldn't it be changed in any other place in the business logic?